docs/plugins/oracledb_metrics.md

```diff
@@ -0,0 +1,67 @@
+# OracleDB Metrics
+
+Metrics receiver for Oracle Databases.
+
+## Supported Versions
+
+- 12.2
+- 18c
+- 19c
+- 21c
+
+## Configuration Parameters
+
+| Name | Description | Type | Default | Required | Values |
```

Comment:
What is the purpose of the Value column if all values are empty?

Reply:
It's for enum typed parameters. I plan on the documentation tool autogenerating docs like this and tried to make this doc as close to the template I envisioned as possible. That way it won't change significantly once we have a generation tool.